lib/components/victory-brush-container.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Rect, G } from "react-native-svg";
+import { last } from "lodash";
```

Comment:
You should change how we're depending on `lodash` in `package.json`. I think prior to this pr we were only depending on `lodash.omit`